src/Microsoft.Framework.CommandLineUtils/CommandLine/AnsiConsole.cs

```diff
@@ -9,22 +9,34 @@ namespace Microsoft.Framework.Runtime.Common.CommandLine
 {
     internal class AnsiConsole
     {
-        private AnsiConsole(TextWriter writer)
+        private AnsiConsole(TextWriter writer, bool useAnsiCodes)
```

Comment:
This name doesn't feel right. You are using ansiCodes either way, just processing them differently. It is more like ansiCodePassthrough, but given that nobody knows what that means and this is internal it doesn't worry me too much.